### DISCUSSION TITLE
The underlying connection was closed: An unexpected error occurred on a send

### DISCUSSION THREAD

**Discussion by ben79git** [pnp/powershell#776] [General]
Hello all, I wonder if someone has seen this before... 

We are having issues connecting to SharePoint via the Connect-PnPOnline command. This has been working perfectly for a while, and I've no idea why it's now not working. We are getting:

Connect-PnPOnline : The underlying connection was closed: An unexpected error occurred on a send

I've checked the TLS settings, which lots of posts point to and they seem all correct.

Any ideas?

**Comment by tristian2** [comment]
BUMP, i get this sporadically on some working powershell, V frustrating

Connect-PnPOnline : The underlying connection was closed: An unexpected error occurred on a send.

**Comment by sdooodly** [comment]
Did you try adding CLientID param?
